packages/ag-charts-community/src/scene/shape/text.ts

```diff
@@ -247,6 +247,10 @@ export class Text<D = any> extends Shape<D> {
     ): void {
         const { lines, x, y } = this;
 
+        if (isNaN(x) || isNaN(y) || isNaN(offsetY) || isNaN(lineHeight)) {
```

Comment:
```suggestion
        if (!Number.isFinite(x) || !Number.isFinite(y) || !Number.isFinite(offsetY) || !Number.isFinite(lineHeight)) {
```

Since I'd assume the canvas won't handle infinite values too

Also just a note that the global isNaN will coerce a string to a number, so `isNaN('f')` returns true - but `Number.isNaN` will only return true for a true NaN value

Reply:
Updated.

Reply:
Change applied, thanks.